account/models.py

```diff
@@ -39,13 +39,15 @@ class CustomUser(AbstractBaseUser, PermissionsMixin):
         verbose_name="email address", max_length=255, unique=True, null=True
     )
     first_name = models.CharField(max_length=25, verbose_name="first name")
-    last_name = models.CharField(max_length=25, verbose_name="first name")
+    last_name = models.CharField(max_length=25, verbose_name="last name")
```

Comment:
Consider using gettext or gettext_lazy for constant strings (verbose name, help, API messages, etc.), as in the future, it would be easier to use Django translation. You might check that to gain more knowledge about multilingual platforms